### PR TITLE
TRUNK-5725 : Bumped up the hibernate-validator

### PR DESCRIPTION
**Pull request by dino-saurabh**
## Description of what I changed

Updated the `org.hibernate:hibernate-validator` from 5.4.3.Final to 6.1.2.Final

## Issue I worked on

see https://issues.openmrs.org/browse/TRUNK-5725

## Checklist: I completed these to help reviewers :)

- [ ] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [ ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [ ] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [ ] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [ ] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

**Comment by LuGO0** [comment]
> @LuGO0 did you also try the latest release of this artifact (hibernate validator)?

No @dkayiwa I did not, I just tried with the version number mentioned in the ticket ID

**Comment by dkayiwa** [comment]
@LuGO0 if your reference application was able to start, then the version of the platform that you run does not have version 7.0.1 of the hibernate validator.

**Comment by LuGO0** [comment]
Do I need to run `mvn openmrs-sdk:build` before `mvn openmrs-sdk:deploy` ?

**Comment by dkayiwa** [comment]
So what did you do to confirm it? 😊

**Comment by dkayiwa** [comment]
Now you are talking 👍

File: pom.xml [hunk 1]
```diff
@@ -235,7 +235,7 @@
 			<dependency>
 				<groupId>org.hibernate</groupId>
 				<artifactId>hibernate-validator</artifactId>
-				<version>5.4.3.Final</version>
+				<version>7.0.1.Final</version>
```
Comment (sherrif10) [inline pom.xml:238]: Is this the latest version of hibernate validator ?

Comment (LuGO0) [reply pom.xml:238]: I have tried almost all the versions with correct group Id, dosent work for me

**Pull request closed without merging**